internal/messagejournal/messagejournal.go

```diff
@@ -0,0 +1,216 @@
+package messagejournal
+
+import (
+	"database/sql"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/redhatinsights/yggdrasil"
+	"github.com/redhatinsights/yggdrasil/ipc"
+)
+
+const messageJournalTemporaryDropString string = `
+	DROP TABLE IF EXISTS runtime;
+`
+const messageJournalTemporaryCreateString string = `
+	CREATE TABLE runtime (
+		id INTEGER NOT NULL PRIMARY KEY,
+		message_id VARCHAR(36) NOT NULL,
+		sent DATETIME NOT NULL,
+		worker_name VARCHAR(128) NOT NULL,
+		response_to VARCHAR(36),
+		worker_event INTEGER,
+		worker_message TEXT
+	);
+`
+const messageJournalPersistentCreateString string = `
```

Comment:
Same comment here.

And one more thing. The command is almost the same as for `messageJournalTemporaryCreateString`. Created table contains same rows. The command differs only in `persistent` keyword. I believe this code deserves some refactoring. :-)